android/src/main/java/io/sentry/react/RNSentryTimeToDisplay.java

```diff
@@ -0,0 +1,26 @@
+package io.sentry.react;
+
+import com.facebook.react.bridge.Promise;
+
+import android.view.Choreographer;
+
+import org.jetbrains.annotations.NotNull;
+import io.sentry.SentryDate;
+import io.sentry.SentryDateProvider;
+import io.sentry.android.core.SentryAndroidDateProvider;
+
+public class RNSentryTimeToDisplay {
+
+    public void GetTimeToDisplay(Promise promise) {
+        Choreographer choreographer = Choreographer.getInstance();
+
+        // Invoke the callback after the frame is rendered
+        choreographer.postFrameCallback(frameTimeNanos -> {
+                final @NotNull SentryDateProvider dateProvider = new SentryAndroidDateProvider();
```

Comment:
Can we reuse the `dateProvider`, maybe even share it with the main Impl.

So we avoid creating it on every call.